hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioInboundPipeline.java

```diff
@@ -51,11 +55,11 @@
     private InboundHandler[] handlers = new InboundHandler[0];
     private ByteBuffer receiveBuffer;
 
-    @Probe(name = "bytesRead")
+    @Probe(name = NETWORKING_METRIC_NIO_INBOUND_PIPELINE_BYTES_READ)
```

Comment:
Shouldn't this one have `unit = BYTES` attribute?

Reply:
Done